plugins/woocommerce/client/admin/client/settings-payments/components/offline-payment-gateways.tsx

```diff
@@ -0,0 +1,141 @@
+/**
```

Comment:
Note this files was based on [payment-gateways.tsx](https://github.com/woocommerce/woocommerce/blob/6e8805fd1547157c63f89ff0e148eca064df476b/plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateways.tsx), will update to latest design when [this PR](https://github.com/woocommerce/woocommerce/pull/52161) is done